examples/aggregator/gcp/metrics-sidecar.yaml

```diff
@@ -0,0 +1,167 @@
+# mostly copied from https://github.com/Stackdriver/stackdriver-prometheus-sidecar/blob/master/kube/full/prometheus.yaml
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: agg-metrics
+---
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRole
+metadata:
+  name: prometheus
+rules:
+  - apiGroups: ['']
+    resources:
+      - nodes
+      - nodes/proxy
+      - services
+      - endpoints
+      - pods
+    verbs: ['get', 'list', 'watch']
+  - apiGroups:
+      - extensions
+    resources:
+      - ingresses
+    verbs: ['get', 'list', 'watch']
+  - nonResourceURLs: ['/metrics']
+    verbs: ['get']
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: prometheus
+  namespace: agg-metrics
+---
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRoleBinding
+metadata:
+  name: prometheus
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: prometheus
+subjects:
+  - kind: ServiceAccount
+    name: prometheus
+    namespace: agg-metrics
+---
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app: prometheus-k8s
+  annotations:
+    prometheus.io/scrape: 'true'
+    prometheus.io/port: '9090'
+    prometheus.io/port2: '9091'
+  name: prometheus-k8s
+  namespace: agg-metrics
+spec:
+  type: LoadBalancer
+  externalTrafficPolicy: Cluster
+  ports:
+    - name: prometheus
+      nodePort: 32388
+      port: 9090
+      protocol: TCP
+      targetPort: 9090
+    - name: sidecar
+      nodePort: 30183
+      port: 9091
+      protocol: TCP
+      targetPort: 9091
+  selector:
+    app: prometheus
+    prometheus: k8s
+  sessionAffinity: None
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: metrics-sidecar
+  namespace: agg-metrics
+  labels:
+    app: metrics-sidecar
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: metrics-sidecar
+  template:
+    metadata:
+      labels:
+        app: metrics-sidecar
+    spec:
+      serviceAccount: prometheus
+      securityContext:
+        runAsUser: 0
+      containers:
+        - name: prometheus
+          # 2.11 seems to be the highest version that is tested.
+          # TO-DO: experiment with newer versions
```

Comment:
`TODO` is the format we use throughout.

Reply:
Thanks. I change the version to v2.18.1 (newest version), and removed the comment.